src/MCPClient/lib/clientScripts/restructureDIPForContentDMUpload.py

```diff
@@ -273,50 +270,22 @@ def getContentdmCollectionFieldInfo(contentdmServer, targetCollection):
     return collectionFieldInfo
 
 
-# Return the dmdSec with the specific ID value. If dublinCore is True, return
-# the <dublincore> child node only.
-def getDmdSec(metsDom, dmdSecId = 'dmdSec_1', dublinCore = True):
-    for node in metsDom.getElementsByTagName('dmdSec'):
-        for k, v in node.attributes.items():
-            if dublinCore and k == 'ID' and v == dmdSecId:
-                # Assumes there is only one dublincore child element.
-                return node.getElementsByTagName('dublincore')[0]
-            else:
-                return node
+def getObjectDirectoryFiles(objectDir):
+    """
+    Get a list of all the files (recursive) in the provided directory.
 
+    "Even though there can be subdirectories in the objects directory,
+    assumes each file should have a unique name." What does this mean?
```

Comment:
Maybe, but the paths returned in ```fileList``` are the full path, so that seems a weird qualifier.  Maybe it's a statement that the parts returned _are_ the full path, and are therefore unique?  It was in the original comments.